packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -276,7 +281,7 @@ export class OdspDocumentStorageService extends OdspDocumentStorageServiceBase {
 
 									return snapshotCachedEntry;
 								});
-
+						let cacheLookUpTime = performance.now() - startTime;
```

Comment:
This should almost always give us zero, as block of code above does not block (does not use await, so it will measure time till first await somewhere under this.epochTracker.get().